lib/scrabble/dictionary.rb

```diff
@@ -1,4 +1,17 @@
 module Scrabble
 	class Dictionary
+
+		attr_accessor
+		def initialize
+
+		end
+
+		def self.valid_word?(word)
+			File.open("./support/dictionary.txt") do |file|
+				file.any? do |line|
+					line.include?(word.upcase)
```

Comment:
I think you need to require an exact match here. Otherwise a sequence of characters that occurs inside a valid word, but isn't a valid word itself, will be counted as valid. e.g. `"sequence".include?("uenc") => true`